app/repository/hbase/HBase.scala

```diff
@@ -2,6 +2,7 @@ package repository.hbase
 
 object HBase {
   val RowKeyDelimiter = "~"
+  val Wildcard = "*"
   val DefaultColumnGroup = "d"
```

Comment:
Still not too convinced with DefaultColumnGroup. The DB conventional name for column group name is ColumnFamily - maybe rename its to 'DefaultColumnFamily' to keep the naming consistent from DB to client